foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -21,9 +21,10 @@ The default direction for a flex container is horizontal, or `row`, <span id='fl
 ~~~
 
 ### Axes
-
 <span id='flex-axes'>No matter which direction you're using, you need to think of your flex-containers as having 2 axes: the main axis and the cross axis. It is the direction of these axes that changes when the `flex-direction` is changed. In _most_ circumstances, `flex-direction: row` puts the main axis horizontal (left-to-right), and `column` puts the main axis vertical (top-to-bottom).</span>
-
+<br>
+<img src="https://samanthaming.gumlet.io/flexbox30/4-flexbox-axes.jpg.gz" alt="main axes and cross axes" title="image Title" width="300" height="300"/>
```

Comment:
If we want to include this image, we should link to the original source as well. Though it might be better to create something specifically for TOP or add an additional resource at the end of the lesson.

If we do keep an image, let's remove the `title` attribute since it's not really accessible and it doesn't add anything in this context.